src/main/scala/outwatch/dom/package.scala

```diff
@@ -26,8 +26,11 @@ package object dom extends Attributes with Tags with HandlerFactories {
   implicit def compositeModifier(modifiers: Seq[VDomModifier]): VDomModifier = IO.pure(CompositeVDomModifier(modifiers))
 
   implicit class ioVTreeMerge(vnode: VNode) {
-    def apply(args: VDomModifier*): VNode = {
-      vnode.flatMap(vnode_ => vnode_(args:_*))
+    def apply(args: VDomModifier*): VNode= {
+      vnode.flatMap {
+        case vtree: VTree => vtree(args: _*)
+        case sn: StringNode => IO.pure(sn)
```

Comment:
That is not what I meant and I think we should not silently drop these modifiers. Instead only VTree should have an apply method.

It can easily happen to construct something like this:

```
    def myNode: VNode = {
      "hans"
    }

    div(myNode(contentEditable := "hans"))
```

Reply:
Additionally, I still think I would be beneficial to let `VTree` know about its dom.Element type, so that this becomes `VTree[Element]`, so you can reason about the type of that node. This would also require exposing this type.

Reply:
Maybe better to leave this as it was in this PR and tackle it again in another one?

Reply:
> It can easily happen to construct something like this and then the modifier is just gone:

Hmm... how would a modifier get applied to a string node? Currently, that results in runtime `NotImplementedError` because of the `???`.

Reply:
Exactly, it should not be possible to apply a modifier to a string node. And this should be enforced by the typesystem, so that apply is not defined on `StringNode`. Of course, a `NotImplementedError` is wrong, too :)

Reply:
> Maybe better to leave this as it was in this PR and tackle it again in another one?

Yeah, makes sense to limit the scope of this PR, I'll revert the change.

Reply:
I think a string node is normally only used within a VNode (e.g. span("hello world")). Maybe, we could make `StringNode extends VDomModifier_` and deal with it internally? Then, there would only be `VTree` extending `VNode_`.

Reply:
> I think a string node is normally only used within a VNode (e.g. span("hello world")). Maybe, we could make StringNode extends VDomModifier_ and deal with it internally? Then, there would only be VTree extending VNode_.

Yes, I though about that and I like that solution, but it breaks usages like this:

```scala
val stringStream: Observable[String]
div(child <-- stringStream)
```

Possible solutions would be to have a conversion from `StringNode` to `VNode` only for the stream case or to disallow this use it and provide a `div(textChild <-- stringStream)`.

An additional argument to make `StringNode` a `VDomModifier` is that currently at the Snabbdom level something like `span("foo")` ends up translated to `{'sel': 'span', children: [{'text': 'foo'}]}` instead of `{'sel': 'span', 'text': 'foo'}`.

Reply:
> Possible solutions would be to have a conversion from StringNode to VNode only for the stream case or to disallow this use it and provide a div(textChild <-- stringStream).

I like the first option more, because it keeps the api for the user. We could just overload `def <--(stream: Observable[String])`.